addons/account/views/report_invoice.xml

```diff
@@ -53,9 +53,9 @@
                         <h2>
                             <span t-if="not proforma"></span>
                             <span t-else="">PROFORMA</span>
-                            <span t-if="o.move_type == 'out_invoice' and o.state == 'posted'">Invoice</span>
-                            <span t-elif="o.move_type == 'out_invoice' and o.state == 'draft'">Draft Invoice</span>
-                            <span t-elif="o.move_type == 'out_invoice' and o.state == 'cancel'">Cancelled Invoice</span>
+                            <span t-if="o.move_type == 'out_invoice' and o.state == 'posted'" t-att-id="posted_invoice">Invoice</span>
```

Comment:
I don't think we need this diff. Maybe id= instead of t-att-id can be done in master.

Reply:
Sorry, i was waiting on the runbot's results to make sure this is not needed anymore, it will be removed.